website/content/docs/commands/auth/enable.mdx

```diff
@@ -85,3 +85,7 @@ flags](/docs/commands) included on all commands.
 
 - `-seal-wrap` `(bool: false)` - Enable seal wrapping for the mount, causing
   values stored by the mount to be wrapped by the seal's encryption capability.
+
+- `-token-type` `(string: "")` - Specifies the type of tokens that should be
```

Comment:
We may want to maintain consistency by making sure that this matches the help text for the flag in the CLI command itself.